app/governance/components/ProposalTable/ProposalTable.tsx

```diff
@@ -0,0 +1,310 @@
+"use client";
+import React, { useEffect, useMemo, useState } from "react";
+import styles from "./ProposalTable.module.scss";
+import { Proposal } from "@/hooks/gov/interfaces/proposal";
+import {
+  formatDate,
+  formatProposalStatus,
+  formatProposalType,
+} from "@/utils/gov/formatData";
+import Text from "@/components/text";
+import Button from "@/components/button/button";
+import Link from "next/link";
+import { useRouter } from "next/navigation";
+import ToggleGroup from "@/components/groupToggle/ToggleGroup";
+import Table from "@/components/table/table";
+import Container from "@/components/container/container";
+import Spacer from "@/components/layout/spacer";
+
+interface TableProps {
+  proposals: Proposal[];
+}
+
+const ProposalTable = ({ proposals }: TableProps) => {
+  const router = useRouter();
```

Comment:
looks like this isn't actually used anywhere

Reply:
router is being used when rows in the Table are clicked.